assets/src/components/detours/detourPanels/detourFinishedPanel.tsx

```diff
@@ -47,15 +47,26 @@ export const DetourFinishedPanel = ({
         </Button>
 
         <h2 className="c-diversion-panel__h2">Directions</h2>
-        <Form.Control
-          as="textarea"
-          value={editableDirections}
-          onChange={({ target: { value } }) => onChangeDetourText(value)}
-          className="flex-grow-1 mb-3"
-          style={{
-            resize: "none",
-          }}
-        />
+        {/* 
+            We need a way to let the form area take up exactly the space of its content
+            (to avoid double scrollbars). We used this approach:
+            https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas
+
+            The result is that the Form.Control has an invisible twin that helps the 
+            wrapper grow to the appropriate size, and then the Form.Control likewise 
+            assumes that space. All formatting between the Form.Control and the ::after
+            pseudo-element must be identical: border, padding, margin, font.
```

Comment:
suggestion(non-blocking): thoughts?
```suggestion
            assumes that space. All styles that affect layout must be identical
            (e.g., `border`, `padding`, `margin`, `font`) between the `<Form.Control/>`
            and the `div.c-autosized-textarea::after` pseudo-element.
```

Reply:
Yeah, that's clearer! More comprehensive / accurate. Thanks